### PR TITLE
Homecoming

### PR DESCRIPTION
**Pull request by burden**
Overhauled the cayman theme to match Elixir School, and Extracurricular. Feedback from @elixirschool/developers is welcome. Thanks.

Desktop preview
screen shot 2017-09-16 at 10 29 07 pm

Mobile preview
screen shot 2017-09-16 at 10 30 32 pm

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,10 @@
-source 'https://rubygems.org'
-gem 'github-pages'
-gem 'jekyll-paginate'
+source "https://rubygems.org"
+ruby RUBY_VERSION
+
+gem "jekyll"
+
+group :jekyll_plugins do
+  gem 'jekyll-assets'
```
Comment (doomspork) [inline Gemfile]: Please use double quotes for consistency 👍

File: _config_prod.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+google_analytics: 'UA-XXXXXX-X'
```
Comment (doomspork) [inline _config_prod.yml]: Could you update this to the one I sent you?

File: jekyll/_posts/2017-09-14-blockquote-example.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
```
Comment (doomspork) [inline jekyll/_posts/2017-09-14-blockquote-example.md]: Could you remove all of these posts?

**Comment by burden** [comment]
@doomspork changes were made

**Pull request merged**